conf/human_lung_cell_atlas.json

```diff
@@ -6,10 +6,10 @@
   "minified_model_dir": "HLCA_reference_model_minified",
   "adata_url": "https://cellxgene.cziscience.com/e/066943a2-fdac-4b29-b348-40cede398e4e.cxg/",
```

Comment:
Needs to be updated to https://cellxgene.cziscience.com/e/9f222629-9e39-47d0-b83f-e08d610c7479.cxg/, the newest version.

Reply:
that's the extended HLCA, we want core. See here: https://cellxgene.cziscience.com/collections/6f6d381a-7701-4781-935c-db10d30de293